spec/system/candidate_signing_up_spec.rb

```diff
@@ -14,11 +14,33 @@
       expect(page).to have_content t('authentication.check_your_email')
     end
 
-    it 'receives an email with a valid magic link' do
-      open_email('april@pawnee.com')
-      current_email.find_css('a').first.click
+    context 'receives an email with a valid magic link' do
+      let(:sign_in_link) { current_email.find_css('a').first }
 
-      expect(page).to have_content 'april@pawnee.com'
+      before do
+        open_email('april@pawnee.com')
+      end
+
+      it 'does sign user in on the first use of the token' do
+        sign_in_link.click
+        expect(page).to have_content 'april@pawnee.com'
+      end
+
+      it 'does not sign user in on the second use of the token' do
+        sign_in_link.click
+
+        Capybara.reset_sessions!
+        sign_in_link.click
+
+        expect(page).not_to have_content 'april@pawnee.com'
+      end
+
+      it 'does not sign user in when the token expiration time has passed' do
+        Timecop.travel(Time.now + 24.hours + 1.second)
```

Comment:
Can we enforce Timecop [safe mode](https://github.com/travisjeffery/timecop#timecopsafe_mode) to avoid leaky time travel?

Reply:
Added to `rails_helper`, assuming that's as good a place as any 🤔